apps/mobile-alphabet/src/app/Components/Menu.tsx

```diff
@@ -37,17 +49,14 @@ export function MenuScreen() {
     }
 
     const {
-        data: { name, name_english, poster },
+        data: { name, name_english, poster, alphabet_cards: alphabetCards },
     } = alphabetData;
 
     return (
         <View>
-            <Text>Name: {name}</Text>
-            <Text>English: {name_english}</Text>
-            <Text>Poster: {poster.name}</Text>
-            <Text>Poster URL: {poster.url}</Text>
+            <Button testID="Detail" title={'Detail'} onPress={() => navigation.push('Detail')} />
```

Comment:
The button's test ID and title should be `linkToDetail` (in whatever casing you have chosen). It's a button that links to the detail screen, not the detail component itself.